gateway/crates/gateway-binary/src/args/std.rs

```diff
@@ -75,17 +78,38 @@ impl Args {
         }
     }
 
-    pub fn config(&self) -> anyhow::Result<Config> {
-        match self.config.as_ref() {
+    fn config(&self) -> anyhow::Result<Config> {
+        let mut config = match self.config.as_ref() {
             Some(path) => {
                 let config = fs::read_to_string(path).context("could not read config file")?;
-                Ok(toml::from_str(&config)?)
+                toml::from_str(&config)?
             }
-            None => Ok(Config::default()),
+            None => Config::default(),
+        };
+        if let Some(ref token) = self.grafbase_access_token {
```

Comment:
So we still need some way of doing this in AWS Lambda?

Reply:
Well yes, not sure how.